lib/Rex/Virtualization/LibVirt/create.pm

```diff
@@ -460,6 +471,11 @@ __DATA__
    <controller type="ide" index="0">
     <address type="pci" domain="0x0000" bus="0x00" slot="0x01" function="0x1"/>
    </controller>
+   <% if (grep { exists($_->{bus}) && $_->{bus} =~ /scsi/ } @{$::storage}) { %>
+   <controller type='scsi' index='0' model='<%= $::scsi_model // 'virtio-scsi' %>'>
```

Comment:
I feel like we should follow something like a "use libvirt defaults, unless otherwise specified" approach around here.

Based on libvirt [docs](https://libvirt.org/formatdomain.html#controllers) and [codebase](https://github.com/libvirt/libvirt/blob/c0ae2ca081c37ce2420d0c500619e989fab9949b/src/vmx/vmx.c#L1253-L1287) I would either:

- leave out the optional `model` attribute if it's undefined
- prefer `auto` instead of `virtio-scsi` if we must provde a value

@alip: what do you think?

Reply:
Going for `auto` is simpler imho and docs say there's no version limitation for the model parameter for the scsi controller. Updated.